packages/cli/src/util/build/static-builder.ts

```diff
@@ -39,3 +40,18 @@ export const build: BuildV2 = async ({ entrypoint, files, config }) => {
 
   return { output };
 };
+
+export const shouldServe: ShouldServe = _opts => {
+  const opts = { ..._opts };
+  const {
+    config: { zeroConfig, outputDirectory },
+  } = opts;
+
+  // Add the output directory prefix
+  if (zeroConfig && outputDirectory) {
+    opts.entrypoint = `${outputDirectory}/${opts.entrypoint}`;
```

Comment:
Should this use `path.join()` so it works with windows?

Reply:
I'd say using `path.join()` is a best practice, but it's not necessary with modern Node.js versions:

```
> const p = process.cwd().replaceAll('\\', '/')
undefined

> p
'C:/Users/chris/projects/vercel/packages'

> fs.readdirSync(p)
[
  'build-utils',   'cli',
  'client',        'edge',
  'frameworks',    'fs-detectors',
  'go',            'hydrogen',
  'next',          'node',
  'node-bridge',   'python',
  'redwood',       'remix',
  'routing-utils', 'ruby',
  'static-build',  'static-config'
]
```